src/components/AboutContent.js

```diff
@@ -10,9 +10,8 @@ const AboutContent = function ({ snippets }) {
   return (
     <div>
 
-      <Header
+      <Header className="ac-header"
         as='h1'
-        style={{ fontSize: '4em' }}>
```

Comment:
You've gotten rid of the closing angle bracket here, which is probably why the build is failing.

Reply:
Good catch, just fixed that